src/nvim/window.c

```diff
@@ -5771,17 +5771,25 @@ void may_trigger_win_scrolled_resized(void)
 
   recursive = true;
 
+  // Save window info before autocmds since they can free windows
+  char resize_winid[NUMBUFLEN];
+  if (trigger_resize) {
+    vim_snprintf(resize_winid, sizeof(resize_winid), "%d", first_size_win->handle);
+  }
```

Comment:
Is the problem that `first_size_win` can be freed, or `w_buffer`? Or both? Anyway, it will be clearer if we save that info here, rather than doing the `vim_snprintf`. 

Something like this:


```suggestion
  handle_T resize_winid = first_size_win->handle;
  handle_T resize_bufid = first_size_win->w_buffer->handle;
```

Then use `resize_winid` and `resize_bufid` in the `if (trigger_resize)` block.